Code/Functions/Public/b.ps1

```diff
@@ -0,0 +1,63 @@
+Function b {
+    <#
+        .SYNOPSIS
+
+        Generates a <b> HTML tag.
+        The <b> tag defines a hyperlink, which is used to link from one page to another.
+        
+        .DESCRIPTION
+
+        .PARAMETER Class
+        Allows to specify one (or more) class(es) to assign the html element.
+        More then one class can be assigned by seperating them with a white space.
+
+        .PARAMETER Id
+        Allows to specify an id to assign the html element.
+
+        .PARAMETER Content
+        Allows to add child element(s) inside the current opening and closing HTML tag(s).
+
+
+        .EXAMPLE
+        The following exapmles show cases how to create an empty a, with a class, an ID, and, custom attributes.
```

Comment:
Shouldn't this be? 
he following exapmles show cases how to create an empty b, ?